examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/termsandconditions/_cq_template/.content.xml

```diff
@@ -19,6 +19,7 @@
     jcr:title="I agree to the terms &amp; conditions"
     sling:resourceType="core/fd/components/form/checkbox/v1/checkbox"
     enabled="false"
+    required="true"
```

Comment:
Please add the changes in archetype proxy also
https://github.com/adobe/aem-project-archetype/blob/develop/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/termsandconditions/_cq_template.xml